kuka_sunrise_fri_driver/src/robot_manager_node.cpp

```diff
@@ -115,6 +126,18 @@ RobotManagerNode::RobotManagerNode() : kuka_drivers_core::ROS2BaseLCNode("robot_
     "joint_damping", joint_damping_, kuka_drivers_core::ParameterSetAccessRights{true, false},
     [this](const std::vector<double> & joint_damping)
     { return this->onJointDampingChangeRequest(joint_damping); });
+
+  registerParameter<std::vector<double>>(
+    "cartesian_stiffness", cartesian_stiffness_,
+    kuka_drivers_core::ParameterSetAccessRights{true, false},
```

Comment:
As you set the stiffness during configuration, it should not be allowed to change the parameters in configured state.
Same applies for joint stiffness and damping
```suggestion
    kuka_drivers_core::ParameterSetAccessRights{false, false},
```